### PR TITLE
more doxygen

### PR DESCRIPTION
**Pull request by edwardhartnett**
More doxygen work.

Part of #31

### CODE REVIEW

File: src/speps.f [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
 C> If L is the zonal wavenumber and N is the total wavenumber
 C> and A is the earth radius, then the fields returned are:
-C> - (1) NORMALIZING FACTOR EPSILON=SQRT((N**2-L**2)/(4*N**2-1))
-C> - (2) LAPLACIAN FACTOR N*(N+1)/A**2
-C> - (3) ZONAL DERIVATIVE/LAPLACIAN FACTOR L/(N*(N+1))*A
-C> - (4) MERIDIONAL DERIVATIVE/LAPLACIAN FACTOR EPSILON/N*A
+C> - (1) normalizing factor EPSILON=SQRT((N**2-L**2)/(4*N**2-1))
+C> - (2) Laplacian factor N*(N+1)/A**2
+C> - (3) zonal derivative/Laplacian factor L/(N*(N+1))*A
+C> - (4) Meridional derivative/Laplacian factor EPSILON/N*A
```
Comment (AlexanderRichert-NOAA) [inline src/speps.f:13]: I don't think "Meridional" needs to be capitalized. That said, I've been mostly been capitalizing variable descriptions and such (just the first word of a sentence/phrase). Do you prefer those to be decapitalized?

File: src/spfftpt.f [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
-C>
-C> Compute fourier transform to gridpoints
-C> @author IREDELL @date 96-02-20
+C> @brief Compute fourier transform to gridpoints.
+C> @author Iredell @date 96-02-20
 
-C> THIS SUBPROGRAM COMPUTES A SLOW FOURIER TRANSFORM
-C> FROM FOURIER SPACE TO A SET OF GRIDPOINTS.
+C> This subprogram computes a slow Fourier transform
+c> from Fourier space to a set of gridpoints.
```
Comment (AlexanderRichert-NOAA) [inline src/spfftpt.f]: c> vs. C>

File: src/spuv2dz.f [hunk 3]
```diff
@@ -1,49 +1,57 @@
...
+C> @param ENN1 real ((M+1)*((I+1)*M+2)/2) N*(N+1)/A**2
+C> @param ELONN1 real ((M+1)*((I+1)*M+2)/2) L/(N*(N+1))*A
+C> @param EON real ((M+1)*((I+1)*M+2)/2) EPSILON/N*A
+C> @param EONTOP real (M+1) EPSILON/N*A over top
+C> @param U real ((M+1)*((I+1)*M+2)) zonal wind (over coslat)
+C> @param V real ((M+1)*((I+1)*M+2)) merid wind (over coslat)
+C> @param UTOP real (2*(M+1)) zonal wind (over coslat) over top
+C> @param VTOP real (2*(M+1)) merid wind (over coslat) over top
+C> @param D real ((M+1)*((I+1)*M+2)) divergence
+C> @param Z real ((M+1)*((I+1)*M+2)) vorticity
```
Comment (AlexanderRichert-NOAA) [inline src/spuv2dz.f]: Still has data types

File: src/spvar.f [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
-C> @param I        - INTEGER SPECTRAL DOMAIN SHAPE
-C>                (0 FOR TRIANGULAR, 1 FOR RHOMBOIDAL)
-C> @param M        - INTEGER SPECTRAL TRUNCATION
-C> @param Q        - REAL ((M+1)*((I+1)*M+2)) SCALAR FIELD
-C> @param QVAR     - REAL (0:(I+1)*M) VARIANCES
+C> @param I integer spectral domain shape
+C> (0 for triangular, 1 for rhomboidal)
+C> @param M integer spectral truncation
+C> @param Q real ((M+1)*((I+1)*M+2)) scalar field
+C> @param QVAR real (0:(I+1)*M) variances
```
Comment (AlexanderRichert-NOAA) [inline src/spvar.f]: Still has data types

File: src/speps.f (hunk 1 above)
Comment (edwardhartnett) [reply src/speps.f:13]: I don't have strong opinions there but we are getting close to the end of this round of sp doxygen improvements, as far as I am concerned. We've made a lot of progress!

File: src/spvar.f (hunk 4 above)
Comment (edwardhartnett) [reply src/spvar.f]: Fixed.

File: src/spfftpt.f (hunk 2 above)
Comment (edwardhartnett) [reply src/spfftpt.f]: Fixed

File: src/spvar.f (hunk 4 above)
Comment (edwardhartnett) [reply src/spvar.f]: Fixed.

**Review by AlexanderRichert-NOAA — APPROVED** [review]

**Pull request merged**